bin/build-release-ccd-definition.sh

```diff
@@ -16,7 +16,7 @@ elif [ ${environment} == local ]; then
 elif [ ${excludeNonProdFiles} == true ]; then
```

Comment:
It should be excluded in both if and else condition as we are not live yet. In my opinion name should be also like "....GAR2GAspec"  to continue with same naming pattern